### PR TITLE
livelike-cloud action destination PE-41

### PR DESCRIPTION
**Pull request by akansal1599**
This is the initial livelike-cloud destination.
This includes an action for tracking events and 3 presets for `track`, `page` and `screen` event types.

_A summary of your pull request, including the what change you're making and why._

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [✓] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/livelike-cloud/trackEvent/generated-types.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+   */
+  user_id?: string
+  /**
+   * The unique LiveLike user identifier. Atleast one of `LiveLike User Profile ID` or `User ID` is mandatory.
+   */
+  livelike_profile_id?: string
+  /**
+   * The timestamp of the event.
+   */
+  timestamp: string | number
```
Comment (joe-ayoub-segment) [inline packages/destination-actions/src/destinations/livelike-cloud/trackEvent/generated-types.ts:27]: type for timestamp should be string only

Comment (joe-ayoub-segment) [reply packages/destination-actions/src/destinations/livelike-cloud/trackEvent/generated-types.ts:27]: Add comment to explain what format the timestamp should be in.

Comment (joe-ayoub-segment) [reply packages/destination-actions/src/destinations/livelike-cloud/trackEvent/generated-types.ts:27]: Add comment to explain what format the timestamp should be in.

File: packages/destination-actions/src/destinations/livelike-cloud/trackEvent/generated-types.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  /**
+   * The name of the action being performed.
+   */
+  action_name?: string
+  /**
+   * The description of the Action.
+   */
+  action_description?: string
+  /**
+   * A unique identifier for a user. Atleast one of `User ID` or `LiveLike User Profile ID` is mandatory.
```
Comment (joe-ayoub-segment) [inline packages/destination-actions/src/destinations/livelike-cloud/trackEvent/generated-types.ts:17]: Spelling correction "At least"

Comment (joe-ayoub-segment) [inline packages/destination-actions/src/destinations/livelike-cloud/trackEvent/generated-types.ts:17]: Consider explaining the difference between the userId and the livelike_profile_id. What values should be passed for each field? How do they behave when sent to Livelike? Any advice you can give the customer?

**Review by joe-ayoub-segment — CHANGES_REQUESTED** [review]

File: packages/destination-actions/src/destinations/livelike-cloud/trackEvent/index.ts [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+      type: 'string',
+      default: {
+        '@path': '$.properties.action_description'
+      }
+    },
+    user_id: {
+      label: 'User ID',
+      type: 'string',
+      description:
+        'A unique identifier for a user. Atleast one of `User ID` or `LiveLike User Profile ID` is mandatory.',
```
Comment (joe-ayoub-segment) [inline packages/destination-actions/src/destinations/livelike-cloud/trackEvent/index.ts:60]:
```suggestion
        'A unique identifier for a user. At least one of `User ID` or `LiveLike User Profile ID` is mandatory.',
```

**Review by joe-ayoub-segment — CHANGES_REQUESTED** [review]

**Comment by joe-ayoub-segment** [comment]
Happy to approve this PR to go to Private Beta. Only changes required before we go to Public Beta are comments related.

**Review by joe-ayoub-segment — APPROVED** [review]
Looks good. New Integration.

**Comment by joe-ayoub-segment** [comment]
Hi @akansal1599 , 
Just FYI the deployment didn't go out today due to an issue unrelated to your PR. It should be deployed later today.

**Comment by joe-ayoub-segment** [comment]
Hi @akansal1599 , 
Unfortunately the deployment yesterday was cancelled (nothing related to your PR). The team plans to release next week. I'll confirm the date when I know more. Apologies for the inconvenience.

**Comment by akansal1599** [comment]
> Hi @akansal1599 , Unfortunately the deployment yesterday was cancelled (nothing related to your PR). The team plans to release next week. I'll confirm the date when I know more. Apologies for the inconvenience.

@joe-ayoub-segment Thanks for the update.

**Comment by joe-ayoub-segment** [comment]
Hi @akansal1599 , 

This PR was deployed yesterday. You should be able to start testing it now!
It's available in Private Beta, so you'll need to access it via a specific URL.
https://app.segment.com/YOUR_WORKSPACE_SLUG/destinations/catalog/actions-livelike-cloud

If you've any follow up questions, including regarding adding meta data and documentation please email [partner-support@segment.com](mailto:partner-support@segment.com).

**Pull request merged**